test/presenters/supergroups/policy_and_engagement_test.rb

```diff
@@ -28,5 +28,78 @@
 
       assert_equal expected, policy_and_engagement_supergroup.document_list(taxon_id)
     end
+
+    it 'prioritises consultations over other content' do
+      tagged_content = []
+      tagged_content.push(*section_tagged_content_list('consultation_outcome'))
+      tagged_content.push(*section_tagged_content_list('case_study'))
+      tagged_content.push(*section_tagged_content_list('open_consultation'))
+      tagged_content.push(*section_tagged_content_list('case_study'))
+      tagged_content.push(*section_tagged_content_list('closed_consultation'))
+
+      MostRecentContent.any_instance
+        .stubs(:fetch)
+        .returns(tagged_content)
+
+      expected = [
+        {
+          link: {
+            text: 'Tagged Content Title',
+            path: '/government/tagged/content'
+          },
+          metadata: {
+            public_updated_at: '2018-02-28T08:01:00.000+00:00',
+            organisations: 'Tagged Content Organisation',
+            document_type: 'Consultation outcome'
+          }
+        },
+        {
+          link: {
+            text: 'Tagged Content Title',
+            path: '/government/tagged/content'
+          },
+          metadata: {
+            public_updated_at: '2018-02-28T08:01:00.000+00:00',
+            organisations: 'Tagged Content Organisation',
+            document_type: 'Open consultation'
+          }
+        },
+        {
+          link: {
+            text: 'Tagged Content Title',
+            path: '/government/tagged/content'
+          },
+          metadata: {
+            public_updated_at: '2018-02-28T08:01:00.000+00:00',
+            organisations: 'Tagged Content Organisation',
+            document_type: 'Closed consultation'
+          }
+        },
+        {
+          link: {
+            text: 'Tagged Content Title',
+            path: '/government/tagged/content'
+          },
+          metadata: {
+            public_updated_at: '2018-02-28T08:01:00.000+00:00',
+            organisations: 'Tagged Content Organisation',
+            document_type: 'Case study'
+          }
+        },
+        {
+          link: {
+            text: 'Tagged Content Title',
+            path: '/government/tagged/content'
+          },
+          metadata: {
+            public_updated_at: '2018-02-28T08:01:00.000+00:00',
+            organisations: 'Tagged Content Organisation',
+            document_type: 'Case study'
+          }
+        }
+      ]
```

Comment:
This expectation is really long. It might be easier to read this test if the expectation was just on the document types that came back, e.g. 

```ruby
expected_document_types = [
  'Consultation outcome',
  'Open consultation',
  'Closed consultation',
  'Case study',
  'Case study'
]

assert_equal(
  expected_document_types, 
  policy_and_engagement_supergroup
    .document_list(taxon_id)
    .map { |x| x['metadata']['document_type'] }
)
```

Reply:
Ah, I see you've done something similar to this here https://github.com/alphagov/collections/pull/626/commits/dda36fa9aca2de83c676db3b3fa189f623707187 :+1: